Misc/NEWS.d/next/Library/2017-09-30-10-45-12.bpo-31648.Cai7ji.rst

```diff
@@ -0,0 +1,4 @@
+Improvements to path predicates in ElementTree:
+* Allow whitespace around predicate parts, i.e. "[a = 'text']" instead of requiring the less readable "[a='text']".
```

Comment:
These lines are merged together in the generated HTML. Add empty lines before and after the enumeration list.